api/users/serializers.py

```diff
@@ -58,7 +58,10 @@ class UserListSerializer(serializers.ModelSerializer):
     join_date = serializers.SerializerMethodField(read_only=True)
 
     default_fields = ("id", "email", "first_name", "last_name", "last_login")
-    organisation_users_fields = ("role", "date_joined")
+    organisation_users_fields = (
+        "role",
```

Comment:
Will this not break if rbac is not installed?

Reply:
This is only a format change, however, this "role" field refers to the "OrganizationRole" model, which is not part of the RBAC.

`class OrganisationRole(models.TextChoices):
    ADMIN = ("ADMIN", "Admin")
    USER = ("USER", "User")`